### PR TITLE
feat: update revoke UI

### PR DESCRIPTION
**Pull request by waynewee**
- updated colors, margins and text

### CODE REVIEW

File: src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx [hunk 1]
```diff
@@ -49,33 +49,36 @@ export const RevokeDocumentDropZone: FunctionComponent<RevokeDocumentDropZone> =
...
                 {documentUploadState === DocumentUploadState.ERROR && (
-                  <div className="max-w-lg text-rose font-bold text-lg" data-testid="error-message">
-                    <p>{errorMessage ? errorMessage : "Error: File cannot be read"}</p>
-                    <p className="text-base text-gray-800 my-4 font-normal">Please try again</p>
+                  <div className="max-w-lg text-rose font-bold text-lg mb-4" data-testid="error-message">
+                    {errorMessage ? errorMessage : "File cannot be read. Please check that you have a valid file"}
                   </div>
                 )}
-                {documentUploadState === DocumentUploadState.INITIALIZED && (
+                {documentUploadState !== DocumentUploadState.LOADING && (
```
Comment (cavacado) [inline src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx:81]: whats wrong with the previous logic?

Comment (waynewee) [reply src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx:81]: this line should show regardless of whether the documentUploadState is in ERROR or INITIALISED, can refer to the figma file

Comment (cavacado) [reply src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx:81]: 👌

**Review by cavacado — APPROVED** [review]
lgtm

**Pull request merged**